Libs/Gtk3/Classes/TextView.cs

```diff
@@ -4,32 +4,60 @@ namespace Gtk
 {
     public partial class TextView
     {
-        public TextView() {}
-
         public static readonly Property<TextBuffer> BufferProperty = Property<TextBuffer>.Register<TextView>(
             Native.BufferProperty,
             nameof(Buffer),
-            (o) => o.Buffer,
+            o => o.Buffer,
             (o, v) => o.Buffer = v
         );
 
-        public TextBuffer Buffer
-        {
-            get => GetProperty(BufferProperty);
-            set => SetProperty(BufferProperty, value);
-        }
-
         public static readonly Property<bool> EditableProperty = Property<bool>.Register<TextView>(
             Native.EditableProperty,
             nameof(Editable),
-            (o) => o.Editable,
+            o => o.Editable,
             (o, v) => o.Editable = v
         );
 
+        public TextView() { }

```

Comment:
Is this coming from me ? 🤔 Anyway let me fix that

Reply:
No it is not, I just stumbled across it. Thank you for fixing :+1:

Reply:
Empty constructors are fine since they propagate via the `GObject(params ConstructParameter[])` constructor (with no parameters provided, it is effectively empty). By design, everything except the IntPtr constructor goes via `g_object_new_with_properties` in the GObject empty base constructor, so just about anything should work for constructor signatures.